webapp/src/app/projects/components/team-overview/team-overview.component.ts

```diff
@@ -18,7 +20,7 @@ export class TeamOverviewComponent implements OnInit, OnDestroy {
   @Select(ProjectsState.currentProject)
   project$: Observable<Project | undefined>;
 
-  @Select(ProjectUsersState.isLoading)
+  @Select(ProjectUsersState.isLoading && ProjectInviteState.isLoading)
```

Comment:
Does this actually work? I think you might have to make two individual selects and combine the observables manually.

Reply:
done